hardware_interface/source/mocks/mock_eps.c

```diff
@@ -68,7 +68,6 @@ static SAT_returnState prv_fill_dummy_data(csp_packet_t *packet) {
                                                .curSolar = 10,
                                                .curBattIn = 10,
                                                .curBattOut = 10,
-                                               .reserved1 = 0,

```

Comment:
I removed some outdated ones. Should I delete the whole mock_eps?

Reply:
@wagnerd97 Are you using these dummy data for HK?

Reply:
I mean some mock tests wouldn't be a bad idea, but being supposedly so close to flatsat I'm not sure the mock testing is worth the time. can always add it back later if needed

Reply:
I think it's worth keeping absolutely. This could be useful for you @wagnerd97, to get some sample data.